Adding Datadog AWS log api endpoints (#4734)
DataDog/documentation: content/en/api/integrations_aws/aws_get_available_log_services.md

---

> title: Get list of AWS log ready services

Reviewer: ```suggestion
title: Get a list of AWS log ready services
```

type: apicontent
order: 15.6
external_redirect: /api/#get-list-of-aws-log-ready-services
---

## Get list of AWS log ready services

<mark>This endpoint is in public beta.</mark>

> Get the list of current AWS services for which Datadog offers automatic log collection.

Reviewer: ```suggestion
Get the list of current AWS services that Datadog offers automatic log collection.
```

> Service IDs returned should be used with the `services` parameter for the [Enable an AWS service log collection][1] API endpoint.

Reviewer: ```suggestion
Use returned service IDs with the `services` parameter for the [Enable an AWS service log collection][1] API endpoint.
```
